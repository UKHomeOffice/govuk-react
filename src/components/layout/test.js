import React from 'react';
import ReactDOM from 'react-dom';
import { Layout } from './';

import { shallow, mount, render } from 'enzyme';

describe('Layout', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Layout>Example</Layout>, div);
  });

});

