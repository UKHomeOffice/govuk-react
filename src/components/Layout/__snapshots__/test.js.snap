// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout matches snapshot: enzyme.mount 1`] = `
.glamor-0,
[data-glamor-0] {
  padding: 10px;
  min-width: 320px;
  box-sizing: border-box;
}

@media only screen and (min-width: 641px) {
  .glamor-0,
  [data-glamor-0] {
    display: -webkit-box;
    display: -moz-box;
    display: -ms-flexbox;
    display: -webkit-flex;
    display: flex;
    flex-direction: column;
    align-items: center;
    -webkit-box-orient: vertical;
    -webkit-box-direction: normal;
    -webkit-flex-direction: column;
    -webkit-box-align: center;
    -webkit-align-items: center;
  }
}

<Layout>
  <glamorous(div)>
    <div
      className="glamor-0"
    >
      example
    </div>
  </glamorous(div)>
</Layout>
`;

exports[`Layout matches snapshot: enzyme.render 1`] = `
.glamor-0,
[data-glamor-0] {
  padding: 10px;
  min-width: 320px;
  box-sizing: border-box;
}

@media only screen and (min-width: 641px) {
  .glamor-0,
  [data-glamor-0] {
    display: -webkit-box;
    display: -moz-box;
    display: -ms-flexbox;
    display: -webkit-flex;
    display: flex;
    flex-direction: column;
    align-items: center;
    -webkit-box-orient: vertical;
    -webkit-box-direction: normal;
    -webkit-flex-direction: column;
    -webkit-box-align: center;
    -webkit-align-items: center;
  }
}

<div
  class="glamor-0"
>
  example
</div>
`;

exports[`Layout matches snapshot: enzyme.shallow 1`] = `
<glamorous(div)>
  example
</glamorous(div)>
`;
